otter/test/worker/test_launch_server_v1.py

```diff
@@ -1156,24 +1163,37 @@ def test_create_server_limits(self):
             'flavorRef': '3'
         }
 
-        ret_ds = [create_server('http://url/', 'my-auth-token', server_config)
+        ret_ds = [create_server('http://url/', 'my-auth-token',
+                                server_config, clock=self.clock)
                   for i in range(3)]
 
-        # no result in any of them and only first 2 treq.post is called
+        # no result in any of them and only 1 treq.post is called
         [self.assertNoResult(d) for d in ret_ds]
```

Comment:
I realize you didn't write this in this PR, but side-effectful LCs are super gross and we should probably turn that into a for loop :)